lib/omedis_web/components/general_components.ex

```diff
@@ -471,7 +471,7 @@ defmodule OmedisWeb.GeneralComponents do
                   <span class="hidden lg:flex lg:items-center">
                     <span class="ml-4 text-sm font-medium leading-6 text-gray-900" aria-hidden="true">
                       <%= if @current_user do %>
-                        <%= @current_user.as_string %>
+                        <%= to_string(@current_user) %>
```

Comment:
`to_string` is called automatically.
Have you tried my proposed solution?
```suggestion
                        <%= @current_user %>
```

Reply:
Oh I see

> In order to promote HTML safety, Phoenix templates
do not use 'Kernel.to_string/1' to convert data types to...

https://hexdocs.pm/phoenix_html/Phoenix.HTML.Safe.html

Then, I guess this is the protocol you should implement instead of the other one.